### ISSUE TITLE
Can a fingerprint recognition function be added? 

### ISSUE THREAD

**Issue by basidangao** [sensepost/gowitness#109]
If possible, hope that the search function will be added to support fingerprint search.

**Comment by basidangao** [comment]
https://github.com/rverton/webanalyze

**Comment by leonjza** [comment]
Yeah I'll add search capabilities for sure. After merging #104 I have some refactoring work I'd like to complete first, then some new features.

It may take some time though.
